Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1304,6 +1304,14 @@ void AXObjectCache::handleChildrenChanged(AccessibilityObject& object)
     else if (auto* axRow = dynamicDowncast<AccessibilityTableRow>(object)) {
         if (auto* parentTable = axRow->parentTable())
             deferRecomputeTableCellSlots(*parentTable);
+    } else if (auto* scrollView = dynamicDowncast<AccessibilityScrollView>(object)) {
+        // When the children of an iframe change, e.g., because it's visibility changes,
```

Comment:
Is it it's or its? I think this may have been my typo, sorry 😅

Reply:
Will fix!